include/coro/net/udp/peer.hpp

```diff
@@ -45,8 +44,8 @@ class peer
 
     peer(const peer&)                             = default;
     peer(peer&&)                                  = default;
-    auto operator=(const peer&) noexcept -> peer& = default;
-    auto operator=(peer&&) noexcept -> peer&      = default;
+    auto operator=(const peer&) noexcept -> peer& = delete;
```

Comment:
I think to match `tcp::client` and `tls::client` the scheduler pointer class field should likely be a raw pointer so the `udp::peer` can be properly copied. It appears I missed converting it in the prior PR that switched from `std::shared_ptr` to `std::unique_ptr`, my bad.

Would you mind switching the PR to match the other types in behavior?

Reply:
Done.